cmd/describe/cluster/cmd_test.go

```diff
@@ -149,6 +151,12 @@ var _ = Describe("getClusterRegistryConfig", func() {
 				DomainName("quay.io").Insecure(true)).
 			PlatformAllowlist(cmv1.NewRegistryAllowlist().ID("test-id"))).Build()
 		Expect(err).NotTo(HaveOccurred())
+		mockCaOutput := ""
+		for registry := range mockCa {
+			mockCaOutput = fmt.Sprintf("%s"+
+				"    - %s: REDACTED\n", mockCaOutput, registry)
+
+		}
```

Comment:
hmm, I meant to use sprintf directly on the map, but you can't do it because of redacted is that it?
I suggest to pull the map keys, sort it then use that list instead of the map to go through the keys and write the redacted alongside it then :thinking: 
Getting the keys via a loop on the map directly results in it being able to be out of order again

Reply:
I see, thanks @gdbranco , let me try again

Reply:
Updated, @gdbranco mind taking another look?